waypoint_planner/src/velocity_set/velocity_set.cpp

```diff
@@ -572,6 +576,19 @@ int main(int argc, char** argv)
   {
     ros::spinOnce();
 
+    geometry_msgs::TransformStamped map_to_lidar_tf;
+
+    try
+    {
+        map_to_lidar_tf = tfBuffer.lookupTransform("map", "lidar", ros::Time::now(), ros::Duration(2.0));
```

Comment:
I'm not sure, but the lider frame is not named as `lidar` but `velodyne`. At least the `wf_simulator` and `base_link_to_localizer` static tf publisher uses `velodyne` frame. Where did you get the lidar frame?

Reply:
Good catch! We use "lidar" on our fork since it is manufacturer-agnostic. I will revert to velodyne.